docs/resources/aws_transit_gateway_multicast_group_source.md

```diff
@@ -0,0 +1,86 @@
+---
+title: About the aws_transit_gateway_multicast_group_source Resource
+platform: aws
+---
+
+# aws\_transit\_gateway\_multicast\_group\_source
+
+Use the `aws_transit_gateway_multicast_group_source` InSpec audit resource to test properties of a single specific AWS EC2 TransitGatewayMulticastGroupSource.
+
+The AWS::EC2::TransitGatewayMulticastGroupSource resource registers sources (network interfaces) with the specified transit gateway multicast domain.
+
+## Syntax
+
+Ensure that transit gateway group source exists.
+
+    describe aws_transit_gateway_multicast_group_source(transit_gateway_multicast_domain_id: "tgw-mcast-domain-1234567890") do
+      it { should exist }
+    end
+
+## Parameters
+
+`transit_gateway_multicast_domain_id` _(required)_
+
+The transit_gateway_multicast_domain_id is the ID of the multicast domain transit gateway.
+
+For additional information, see the [AWS documentation on AWS EC2 TransitGatewayMulticastGroupSource](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-ec2-transitgatewaymulticastgroupsource.html).
+
+## Properties
+
+| Property | Description|
+| --- | --- |
+| group_ip_address | The IP address of the transit gateway multicast group. |
+| transit_gateway_attachment_id | The attachment id of the transit gateway. |
+| subnet_id | The ID of the subnet. |
+| resource_id | The ID of the resource. |
+| resource_type | The type of the resource. |
+| resource_owner_id | The Owner ID of the resource. |
```

Comment:
Should `resource_owner_id` be in here? Also, I don't see `TransitGatewayMulticastDomainId` in the properties.

Reply:
It should be resource_owner_id.
And as this API don't have a TransitGatewayMulticastDomainId property so it is not mentioned over here.

If the user wants to check whether the TransitGatewayMulticastDomainId is present or not then he should use the example as 

describe aws_transit_gateway_multicast_group_source(transit_gateway_multicast_domain_id: "tgw-mcast-domain-1234567890") do
      it { should exist }
    end

Please see the docs https://docs.aws.amazon.com/sdk-for-ruby/v3/api/Aws/EC2/Client.html#search_transit_gateway_multicast_groups-instance_method